.github/workflows/benchmark.yml

```diff
@@ -90,8 +100,31 @@ jobs:
           echo "Benchmark PID: $pid"
           wait $pid
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+
+      - name: Install the project
+        run: uv sync --locked --all-extras --dev
+
       - name: Upload benchmark logs
         uses: actions/upload-artifact@v4
         with:
           name: bench-logs
           path: logs/*.log
+
+      - name: Run memory profiling
+        run: |
+          LOG=logs/data_${{ github.event.inputs.data_id }}_param_${{ github.event.inputs.param_id }}.log
```

Comment:
where are these data coming from?

Reply:
you can insert 
<img width="1185" alt="Screenshot 2025-05-06 at 10 53 57 AM" src="https://github.com/user-attachments/assets/28b71714-62bb-415c-9d15-958e358f142e" />